apps/web-client/src/routes/inventory/inbound/+page.svelte

```diff
@@ -160,16 +155,21 @@
 	{/if}
 </InventoryManagementPage>
 
-{#if $open}
-	{@const { onConfirm, title, description } = dialogContent};
-
-	<div use:melt={$portalled}>
-		<div use:melt={$overlay} class="fixed inset-0 z-50 bg-black/50" transition:fade|global={{ duration: 100 }}></div>
-		<div class="fixed left-[50%] top-[50%] z-50 flex max-w-2xl translate-x-[-50%] translate-y-[-50%]">
-			<Dialog {dialog} type="delete" {onConfirm}>
-				<svelte:fragment slot="title">{title}</svelte:fragment>
-				<svelte:fragment slot="description">{description}</svelte:fragment>
-			</Dialog>
-		</div>
-	</div>
-{/if}
+<PageCenterDialog
+	{dialog}
+	on:cancel={() => {
+		open.set(false);
+		noteToDelete = null;
+	}}
+	description={$LL.common.delete_dialog.title({ entity: noteToDelete?.displayName })}
+	title={$LL.common.delete_dialog.description()}
+>
+	<ConfirmDialog
+		on:confirm={() => {
+			handleDeleteNote(noteToDelete.id);
+			noteToDelete = null;
+		}}
+		on:cancel={() => open.set(false)}
```

Comment:
You forgot to add `noteToDelete = null` here. Write a function once and re-use it. Same applies in `outbound/+page.svelte`